### ISSUE TITLE
is this plugin supports the cakephp 3.0 or greater one 

### ISSUE THREAD

**Issue by ashishdevswami** [ichikaway/cakephp-mongodb#137]
I am using Ichikaway for the cakephp 2.3.7, and about to migrate the cakephp 3.2 so my concern is if ichikaway is supported by the cakephp 3.2.0 or any greter version.

**Comment by suhaboncukcu** [comment]
+1 I'm waiting for the answer too.

**Comment by davehensley** [comment]
No, CakePHP 3.x uses a completely different (and incompatible) ORM, so unfortunately this plugin won't work with it.

**Comment by suhaboncukcu** [comment]
and for the people who will definitely come here to see the answer; according to slack channel chat, core developers had some bad experiences with mongo in past and not planning to implement something for mongodb. They suggest to use elasticsearch (which have an official plugin) and postgres. 

Info on readme would be nice though 👍

**Comment by burzum** [comment]
@suhaboncukcu I would like to add that Elastic Search is *not* thought to be used as a primary DB but as a search index. In my opinion ES is *not* a replacement for MongoDB.

https://www.quora.com/Why-shouldnt-I-use-Elasticsearch-as-my-primary-datastore

However, since both are similar somebody could take a look at the ES plugin to see how it is done and implement a Mongo plugin based on it.
